<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="RightWidget" native="true">
   <property name="geometry">
    <rect>
     <x>829</x>
     <y>0</y>
     <width>455</width>
     <height>720</height>
    </rect>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>151</x>
      <y>61</y>
      <width>260</width>
      <height>469</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="StrategyLabel">
         <property name="text">
          <string>策略选择：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="StrategyComboBox">
         <item>
          <property name="text">
           <string>费用最少</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>耗时最短</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>限时费用最少</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="StartLabel">
         <property name="text">
          <string>始发地：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="StartComboBox">
         <item>
          <property name="text">
           <string>北京</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>上海</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>西安</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>武汉</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>深圳</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>郑州</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>海南</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>拉萨</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>纽约</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>首尔</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>哈尔滨</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>莫斯科</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="DestinationLabel">
         <property name="text">
          <string>目的地：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="DestinationComboBox">
         <item>
          <property name="text">
           <string>北京</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>上海</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>西安</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>武汉</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>深圳</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>郑州</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>海南</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>拉萨</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>纽约</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>首尔</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>哈尔滨</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>莫斯科</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="TimeLabel">
         <property name="text">
          <string>当前时间：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="StartDateTimeEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="DeadlineLabel">
         <property name="text">
          <string>截止时间：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="DeadlineDateTimeEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>138</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="resetButton">
         <property name="minimumSize">
          <size>
           <width>59</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>59</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>重置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="StartButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>59</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>开始</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="SolutionLabel">
         <property name="text">
          <string>解决方案：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="SolutionTextEdit">
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="FareLabel">
         <property name="text">
          <string>预计费用：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="FareEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="TotalTimeLabel">
         <property name="text">
          <string>预计时间：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="TotalTimeEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="DurationLabel">
         <property name="text">
          <string>已用时间：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="DurationText"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="LeftWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>825</width>
     <height>720</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
